uint8				   robot_side
val_ik_msgs/RobotState robot_state
geometry_msgs/Pose 	   des_hand_pose
--- 
val_ik_msgs/RobotState robot_state